### PR TITLE
Blog: Use Scale Jobs For Long Running Workloads

### PR DESCRIPTION
**Pull request by sharadregoti**

### CODE REVIEW

**Comment by sharadregoti** [comment]
Pending tasks:
- Need to build & change docker repository

**Review by zroubalik — COMMENTED** [review]
Looking good, could you please add a short Readme? 
- paragraph what is the blog about
- what is in directories

Also why is the consumer direcory named `archived-`?

File: blogs/keda_scaled_job_for_long_running_workload/readme.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+│   # Contain rabbitmq operator cluster config
+├── rabbitmq_cluster.yaml
+│
+│   # Contain Keda scaled job with amqp protocol config
+├── rabbitmq_scaled_job_amqp.yaml
+│
+│   # Contain Keda scaled object with amqp protocol config
+├── rabbitmq_scaled_object_amqp.yaml
+|
+│   # Contain Keda scaled object with http protocol config
```
Comment (zroubalik) [inline blogs/keda_scaled_job_for_long_running_workload/readme.md:47]:
```suggestion
│   # Contain KEDA scaled job with amqp protocol config
├── rabbitmq_scaled_job_amqp.yaml
│
│   # Contain KEDA scaled object with amqp protocol config
├── rabbitmq_scaled_object_amqp.yaml
|
│   # Contain KEDA scaled object with http protocol config
```

**Comment by sharadregoti** [comment]
Changed directory structure & added readme. 

The consumer directory was prefixed `archived-` as it was not used in the blog post. But was used for local testing & POC

If no further changes appropriate changes need to be done in blog commands as well.

**Review by zroubalik — APPROVED** [review]
LGTM

**Comment by josefkarasek** [comment]
Hey @sharadregoti, I published the docker images in this PR here:
```
ghcr.io/kedify/blog05-cli-consumer-program:latest
ghcr.io/kedify/blog05-python-consumer-program:latest
ghcr.io/kedify/blog05-python-producer-program:latest
ghcr.io/kedify/blog05-result-analyzer-program:latest
```

**Comment by zroubalik** [comment]
@sharadregoti please update manifests

**Comment by sharadregoti** [comment]
Changed done and tested. Working fine

**Review by zroubalik — APPROVED** [review]
LGTM, great job!

**Pull request merged**